ports/esp32s2/boards/unexpectedmaker_tinys2/mpconfigboard.mk

```diff
@@ -0,0 +1,24 @@
+USB_VID = 0x303A
+USB_PID = 0x8002
+USB_PRODUCT = "TinyS2S2"
```

Comment:
do you mean TinyS2 ?

Reply:
Yeah, whoops, fixed! How embarrassing ;)